schemas/test/en/test_list_collector_repeating_blocks.json

```diff
@@ -0,0 +1,258 @@
+                  ]
+                },
+                "answers": [
+                  {
+                    "id": "list-collector-next-question-answer-1",
+                    "mandatory": true,
+                    "type": "Radio",
+                    "options": [
+                      {
+                        "label": "Yes",
+                        "value": "Yes",
+                        "action": {
+                          "type": "RedirectToListAddBlock"
+                        }
+                      },
+                      {
+                        "label": "No",
+                        "value": "No"
+                      }
+                    ]
+                  }
+                ]
+              },
+              "add_block": {
+                "id": "list-collector-add-block",
+                "type": "ListAddQuestion",
+                "question": {
+                  "id": "list-collector-add-block-question",
+                  "type": "General",
+                  "title": "What is the grocery item?",
+                  "guidance": {
+                    "contents": [
+                      {
+                        "description": "This item will be followed up by 2 pages of related questions (repeating blocks)"
+                      }
+                    ]
+                  },
+                  "answers": [
+                    {
+                      "id": "list-collector-add-block-question-answer",
+                      "label": "Name of grocery item",
+                      "mandatory": true,
+                      "type": "TextField"
+                    }
+                  ]
+                }
+              },
+              "repeating_blocks": [
+                {
+                  "id": "repeating-block-1",
+                  "type": "ListRepeatingQuestion",
+                  "question": {
+                    "id": "repeating-block-1-question",
+                    "type": "General",
+                    "title": "Enter details about your grocery item.",
+                    "guidance": {
+                      "contents": [
+                        {
+                          "description": "This is the question for repeating block 1, which relates to an item that was added to the list collector."
+                        },
+                        {
+                          "description": "Enter numbers"
+                        }
+                      ]
+                    },
+                    "answers": [
+                      {
+                        "id": "repeating-block-1-question-answer-1",
+                        "label": "How many grams of this item do you need?",
+                        "mandatory": true,
+                        "type": "Number"
+                      },
+                      {
+                        "id": "repeating-block-1-question-answer-2",
+                        "label": "How much  are you willing to spend on this item?",
```

Comment:
Changes to this are great, only (extremely minor) suggestion would be to change type of this one to currency

Reply:
Changed one to grams and one to currency 👍